base_report_docx/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © <2018> Elico Corp (https://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Base Report Docx',
+    'version': '10.0.1.0.0',
+    'summary': 'Base Report Docx',
+    'category': 'report',
+    'depends': [
```

Comment:
@Kevin4577 I guess you'll need soon or later add Python external dependency if want Travis pass.
And include on .travis.yml file.
And include on README.rst file so the manager could know why the module can't be installed

Reply:
fixed is that ok?

Reply:
Might be but not depend on me. Please make Travis green. Ask for support if you need